### PR TITLE
Reacting to new CoreFx packages

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @natemcmaster

File: makefile.shade [hunk 1]
```diff
@@ -6,14 +6,12 @@ var AUTHORS='Microsoft Open Technologies, Inc.'
 use-standard-lifecycle
 k-standard-goals
 
-  
 #initialize-tools target='initialize'
   @{
     // workaround bug in dotnet-build for netcore50 on .Tools and .Tools.Core
     string restoreOptions = " --infer-runtimes " + E("KOREBUILD_DOTNET_RESTORE_OPTIONS");
   }
   dotnet-restore restore_options='${ restoreOptions } src/Microsoft.EntityFrameworkCore.Tools/'
```
Comment (natemcmaster) [inline makefile.shade]: So we still need `--infer-runtimes` for Ms.EfCore.Tools? That's unfortunate

Comment (pranavkm) [inline makefile.shade]: Yeah, it fails to pack without it :crying_cat_face:

**Comment by natemcmaster** [comment]
:shipit:

**Pull request merged**